### ISSUE TITLE
restart followed by reload is unnecessary

### ISSUE THREAD

**Issue by moofish32** [johnbellone/consul-cookbook#438]
The default recipe currently sets up a situation in which restart is called followed immediately by reload. Is there a benefit here? The disadvantage is that in RHEL 6.9 consul actually fails to start because these are called back to back for me. I would say it's definitely a race condition but seems to be about 8 out of 10 so far. 

If we make this a restart, then only one action is taken and I think everything works.
https://github.com/johnbellone/consul-cookbook/blob/master/recipes/default.rb#L21

**Comment by legal90** [comment]
@moofish32 Thank you for pointing on this! You're right - we actually have two subsequent actions with Consul service on the initial bootstrap: 

```
       Recipe: consul::default
         * consul_service[consul] action restart
           * poise_service[consul] action restart
             * service[consul] action restart
        - restart service service[consul]


         * consul_service[consul] action reload
           * poise_service[consul] action reload
             * service[consul] action reload
        - reload service service[consul]
```
And that should be fixed. I'll look into it soon.

**Comment by legal90** [comment]
Fixed by #433
